optimum/neuron/utils/hub_neuronx_cache.py

```diff
@@ -167,6 +212,10 @@ def upload_file(self, cache_path: str, src_path: str):
         # Only upload to the default cache: use synchronize to populate the Hub cache
         self.default_cache.upload_file(cache_path, src_path)
 
+    def upload_folder(self, cache_dir: str, src_dir: str):
+        # Upload folder to the default cache: use synchronize to populate the Hub cache
+        shutil.copytree(src_dir, cache_dir, dirs_exist_ok=True)
```

Comment:
nit: maybe give the method another name if it does no upload anything? Or you are overriding an existing method?

Reply:
It's the `upload_file` method for traced inference, so I tried to name them in a similar way, `upload_file` in `libneuronxla` also copy file from one path to another like `upload_folder` here.